Antitrust-Policy.md

```diff
@@ -0,0 +1,64 @@
+# CNOE Antitrust Policy 
+
+## Introduction
+Cloud Native Operational Excellence (“CNOE”) is an alliance formed to provide recommended best practices for internal developer platforms utilizing cloud-native open source software (all such recommendations, the “Recommended Best Practices”).  CNOE is guided by the governing principles of being open, being welcoming and respectful, being transparent and accessible and driven by technical merit (“Guiding Principles”). It is CNOE’s policy to comply with all applicable state and federal antitrust and trade regulation laws, and applicable antitrust and competition laws of other countries (collectively, the “Antitrust Laws”).  
+
+This Policy applies to members of CNOE (“Members”) and all participants (“Participants”) as well as their employees, consultants, directors and officers.  All Members and Participants shall comply with all applicable rules, policies and procedures adopted by CNOE or the applicable working group, as updated from time to time.
```

Comment:
Not sure if I understand the distinction between participants and members. is "participants" meant to imply the broader group of users (who may not belong to a particular member company) engaged with the work done in CNOE? if so, the other occurrences of the term "users" should be replaced with the term "participants".